static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -38,6 +38,7 @@ export type OnboardingLayoutProps = {
   projectId: Project['id'];
   projectSlug: Project['slug'];
   activeProductSelection?: ProductSolution[];
+  cdn?: string;
```

Comment:
I see references to Loader below, is is the CDN or the Loader link?

Loader is custom for the user. CDN is a static JS file:

CDN (now a heading in the Loader page): https://docs.sentry.io/platforms/javascript/install/loader/#cdn
Loader: https://docs.sentry.io/platforms/javascript/install/loader/#using-the-loader